cli/azd/cmd/show.go

```diff
@@ -165,7 +178,82 @@ func (s *showAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 		}
 	}
 
-	return nil, s.formatter.Format(res, s.writer, nil)
+	if s.formatter.Kind() == output.JsonFormat {
+		return nil, s.formatter.Format(res, s.writer, nil)
+	}
+
+	appEnvironments, err := s.envManager.List(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	uxEnvironments := make([]*ux.ShowEnvironment, len(appEnvironments))
+	for index, environment := range appEnvironments {
+		uxEnvironments[index] = &ux.ShowEnvironment{
+			Name:      environment.Name,
+			IsCurrent: environment.Name == environmentName,
+			IsRemote:  !environment.HasLocal && environment.HasRemote,
+		}
+	}
+
+	uxServices := make([]*ux.ShowService, len(res.Services))
+	var index int
+	for serviceName, service := range res.Services {
+		uxServices[index] = &ux.ShowService{
+			Name:      serviceName,
+			IngresUrl: service.IngresUrl,
+		}
+		index++
+	}
+
+	s.console.MessageUxItem(ctx, &ux.Show{
+		AppName:         s.azdCtx.GetDefaultProjectName(),
```

Comment:
We should be reading from `projectConfig`/ azure.yaml for this value.

Reply:
Even though the `name` field in `azure.yaml` is set as mandatory within the schema, azd won't return an error if it is missing.  I need to follow up about that field to then decide what to do here